backend/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "build": "tsc",
     "start": "node --experimental-modules --es-module-specifier-resolution=node dist/index.js",
-    "dev": "concurrently \"tsc -w\" \"nodemon --experimental-modules --es-module-specifier-resolution=node dist/index.js\""
+    "dev": "npm run build && concurrently \"tsc -w\" \"nodemon --experimental-modules --es-module-specifier-resolution=node dist/index.js\""
```

Comment:
This is the reason why `npm run backend` would error the first time but be okay the second time. 
By adding the build command at the beginning, it works good the first time.